src/Tribe/Commerce/PayPal/Orders/Report.php

```diff
@@ -39,11 +39,11 @@ class Tribe__Tickets__Commerce__PayPal__Orders__Report {
 	 * @return string The absolute URL.
 	 */
 	public static function get_tickets_report_link( $post ) {
-		$url = add_query_arg( array(
+		$url = add_query_arg( [
 			'post_type' => $post->post_type,
 			'page'      => self::$orders_slug,
 			'post_id'   => $post->ID,
-		), admin_url( 'edit.php' ) );
+		], admin_url( 'edit.php' ) );
```

Comment:
* :no_entry_sign: Only one argument is allowed per line in a multi-line function call
* :no_entry_sign: Closing parenthesis of a multi-line function call must be on a line by itself

<sub>hash: 4fcb4dad6f328b3e7567c8f55ef1e937</sub>